client/disputes/evidence/index.js

```diff
@@ -133,28 +134,53 @@ export const DisputeEvidenceForm = ( props ) => {
 			isLoading,
 			isDone,
 			error,
-			help: expandHelp( field.description ),
+			help: readOnly ? '' : expandHelp( field.description ),
 		};
 	};
 
 	const composeFieldControl = ( field ) => {
 		switch ( field.type ) {
 			case 'file':
-				return (
+				return ! readOnly ? (
 					<FileUploadControl
 						key={ field.key }
 						{ ...composeFileUploadProps( field ) }
 					/>
+				) : (
+					<UploadedReadOnly
+						key={ field.key }
+						{ ...composeFileUploadProps( field ) }
+					/>
 				);
 			case 'text':
-				return (
+				return readOnly && ! evidence[ field.key ] ? (
```

Comment:
I think there's scope to simplify this. While there's no problem with the code, I generally find reading conditions (especially the ternary operator ones) in JSX harder and verbose. We could instead use conditional props. For example:

```javascript
<TextControl
	key={ field.key }
	label={ field.label }
	value={  readOnly ? __(
		'No information submitted',
		'woocommerce-payments'
	) : null }
	disabled={ readOnly }
    { ...( readOnly ? composeDefaultControlProps( field ) : {} ) }
/>
```

Reply:
Reference - https://www.geeksforgeeks.org/how-to-conditionally-add-attributes-to-react-components/